src/libsync/wordlist.cpp

```diff
@@ -0,0 +1,2097 @@
+#include "wordlist.h"
+#include <openssl/rand.h>
+
+namespace OCC {
+namespace WordList {
+
+#include <initializer_list>
+
+int getRandomNumber(int max) {
+    unsigned char d[8];
+    RAND_bytes(d, 8);
```

Comment:
We should also check the return status here, could be 0 in case of an error (e.g. insufficient entropy) - https://www.openssl.org/docs/man1.0.2/crypto/RAND_bytes.html